.byebug_history

```diff
@@ -0,0 +1,3 @@
+exit
+::Spree.solidus_gem_version
+::Spree.solidus_gem_version > Gem::Version.new('2.10')
```

Comment:
This shouldn't be committed.

Reply:
Absolutely, Thanks for pointing that out